docs/pages/blog/migration-update.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Why you should migrate to Material UI v5 today
+description: We have completely revamped our Migration guide to reduce friction when upgrading to v5. Get started now!
+date: 2022-06-10T00:00:00.000Z
+authors: ['samuelsycamore']
+card: true
+tags: ['News', 'MUI Core']
+---
+
+Are you still using Material UI v4 in 2022?
+
+What are you thinking?! 😛
+
+We released v5 in late 2021, and since then, we've seen a steady migration of users over to the new packages.
+This is exciting for us, because we're super proud of all the improvements that were shipped with v5.
+
+But it seems as though a fair share of users are still hesitant to make the jump.
+This could be because v5 contains many breaking changes from v4, so the migration can be a large undertaking.
+
+That's why we've completely revamped our [v4 -> v5 migration guide](https://mui.com/material-ui/migration/migration-v4/)—to help reduce the amount of friction you might encounter when updating to v5.
+
+We hope it helps!
+Be sure to check out [the updated migration guide](https://mui.com/material-ui/migration/migration-v4/) in the Material UI docs.
+
+## Why you should upgrade to v5
```

Comment:
I really like this highlight of the top reasons why people should migrate 👌

Reply:
I wonder if we should make a section for what's coming next, like CSS variables etc. Maybe something like: "You don't want to miss the cool upcoming features we are working on, like support for CSS variables that should enable perfect dark mode etc."

Reply:
Good call! @siriwatknp has suggested replacing one of my list items with this, and I think that's a great addition.